bin/bundle

```diff
@@ -28,7 +28,7 @@ m = Module.new do
       if update_index && update_index.succ == i && a =~ Gem::Version::ANCHORED_VERSION_PATTERN
         bundler_version = a
       end
-      next unless a =~ /\A--bundler(?:[= ](#{Gem::Version::VERSION_PATTERN}))?\z/
+      next unless a =~ /\A--bundler(?:[= ](#{Gem::Version::VERSION_PATTERN}))?\z/o
```

Comment:
What are these "o" characters at the end?

Reply:
Hey @joemasilotti - it came as a suggestion from Standardrb.

This was the message on terminal: ```bin/bundle:31:24: Performance/ConstantRegexp: Extract this regexp into a constant, memoize it, or append an `/o` option to its options.```